### DISCUSSION TITLE
Opening duplicate nav items

### DISCUSSION THREAD

**Discussion by jackmsvaughan** [markdoc/markdoc#143] [Q&A] [answered]
Occasionally it's useful to have the same article in 2 places.

Currently, we can do this — but as we're linking to the same file the nav will jump to the article in its original location. It would be great if the nav stayed open on whichever one the user clicked (and for us to not have to create a duplicate of that article in the files). 

Here's the current unwanted behavior. Does anyone have any tips on how to improve this?

https://user-images.githubusercontent.com/50674127/180646003-18dda952-814b-4800-82d6-070cdc26a7b8.mp4

**Comment by tonglil** [comment]
Sorry to be unrelated, but how did you get more levels in the navigation?

**Comment by mfix-stripe** [comment, marked as answer]
I'm assuming you two are both using the Markdoc starter? If so, you can just edit the code here: https://github.com/markdoc/markdoc-starter/blob/main/components/SideNav.tsx

This line determines active item, and you can open the corresponding section accordingly: https://github.com/markdoc/markdoc-starter/blob/main/components/SideNav.tsx#L22

If you want deeper levels of nesting, you just need to add a new `ul` to the `li` here: https://github.com/markdoc/markdoc-starter/blob/main/components/SideNav.tsx#L25
